tabpfn_client/client.py

```diff
@@ -88,7 +87,7 @@ def upload_train_set(self, X, y) -> str:
         train_set_uid = response.json()["train_set_uid"]
         return train_set_uid
 
-    def predict(self, train_set_uid: str, x_test):
+    def predict(self, train_set_uid: str, x_test, tabpfn_config: dict = None):
```

Comment:
I think it would be better to change the API s.t. you only have a predict_proba endpoint, as this really is the superset of predict. Then we could save a lot of complication especially in the layers of this PR.

Reply:
Agreed.

I kept `predict` instead of `predict_proba` though, would that be okay? Just a slight preference of mine (we could also change this) Rationale being: calling `predict_proba(with_proba=false)` seems strange (like ordering cheese burger but without cheese) :D